connect/cli/plugins/locale/commands.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+# This file is part of the Ingram Micro Cloud Blue Connect connect-cli.
+# Copyright (c) 2019-2022 Ingram Micro. All Rights Reserved.
+
+import click
+from cmr import render
+
+from connect.cli.core import group
+from connect.cli.core.config import pass_config
+from connect.cli.core.utils import continue_or_quit
+from connect.cli.plugins.locale.constants import LOCALES_TABLE_HEADER
+from connect.cli.plugins.locale.utils import (
+    icon_for_autotranslate,
+    row_format_resource,
+    stats_of_translations,
+)
+from connect.client import ConnectClient, RequestLogger
+
+
+@group(name='locale', short_help='List all locales avaibles.')
+def grp_locales():
+    pass  # pragma: no cover
+
+
+@grp_locales.command(
+    name='list',
+    short_help='List locales.',
+)
+@click.option(
+    '--query',
+    '-q',
+    'query',
+    help='RQL query expression.',
+)
+@click.option(
```

Comment:
I would suggest to use IntRange here https://click.palletsprojects.com/en/8.1.x/api/#click.IntRange

Reply:
Thanks for the advice, is updated to use `click.IntRange` for integer validation at `page_size` option now